exonum-java-binding-core/rust/integration_tests/tests/utils_errors.rs

```diff
@@ -157,8 +186,14 @@ fn get_and_clear_java_exception_if_no_exception_occurred() {
         .unwrap();
 }
 
-fn throw(env: &JNIEnv, e: &str) -> JniResult<()> {
-    env.throw((e, ""))?;
+fn throw(env: &JNIEnv, exception_class: &str) -> JniResult<()> {
+    // FIXME throw an exception without a stub message https://jira.bf.local/browse/ECR-1998
+    env.throw((exception_class, ""))?;
```

Comment:
Would it be possible to

1 Create an exception using no-arg constructor using env.new_object?
2 Throw it with env.throw?

Reply:
2. Currently it is impossible with jni-rs 😞

Reply:
That's weird, I thought a reference to a throwable object works with signature `Desc<Throwable>`: https://docs.rs/jni/0.10.1/jni/struct.JNIEnv.html#method.throw

Reply:
I found implementations of Desc<JThrowable> only here: https://github.com/prevoty/jni-rs/blob/master/src/wrapper/descriptors/exception_desc.rs

All of them imply a pair of:
- an object with a class name or `java/lang/Exception` as a default, and
- an argument, castable to a string.

Reply:
I see, thanks!